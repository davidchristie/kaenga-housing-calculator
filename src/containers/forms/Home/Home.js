import { reduxForm } from 'redux-form'

import Home from '../../../components/forms/Home'

export default reduxForm({
  form: 'home'
})(Home)
