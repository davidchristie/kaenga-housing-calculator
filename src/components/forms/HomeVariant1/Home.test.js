import React from 'react'
import ReactTestUtils from 'react-addons-test-utils'

import Home from './Home'

const shallowRenderer = ReactTestUtils.createRenderer()

it('matches snapshot', () => {
  expect(shallowRenderer.render(<Home />)).toMatchSnapshot()
})
