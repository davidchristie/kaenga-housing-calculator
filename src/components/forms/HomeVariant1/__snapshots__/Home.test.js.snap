exports[`test matches snapshot 1`] = `
<Connect(Form(Home))
  destroyOnUnmount={true}
  enableReinitialize={false}
  forceUnregisterOnUnmount={false}
  form="homeVariant1"
  getFormState={[Function]}
  initialValues={undefined}
  keepDirtyOnReinitialize={false}
  persistentSubmitErrors={false}
  pure={true}
  shouldAsyncValidate={[Function]}
  shouldValidate={[Function]}
  touchOnBlur={true}
  touchOnChange={false} />
`;
